chardet/charsetprober.py

```diff
@@ -79,8 +78,8 @@ def filter_international_words(buf):
 
         This filter applies to all scripts which do not use English characters.
         """
-        filtered = BytesIO()
 
+        out = b''
```

Comment:
Speed is approx. same, but ``filtered.getvalue()`` changes 'bytearray' to 'str' in the python 2.7 and it makes problems in probers. If you know clear solution without ``compat.py`` please send me info...